tornado/simple_httpclient.py

```diff
@@ -209,6 +209,7 @@ def _on_timeout(self, key, info=None):
 
 class _HTTPConnection(httputil.HTTPMessageDelegate):
     _SUPPORTED_METHODS = set(["GET", "HEAD", "POST", "PUT", "DELETE", "PATCH", "OPTIONS"])
+    _SAFE_NETHODS = set(["GET", "HEAD", "OPTIONS"])
```

Comment:
minor typo: `N` should be `M` in "NETHODS"

Reply:
Thanks for your reminding. I‘m too careless.